CHANGELOG.md

```diff
@@ -5,33 +5,6 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [2.2.0] - 2023-07-25
```

Comment:
revert please (and commit with `git commit --amend --no-edit` to ensure things stay squashed)